example/src/containers/index.jsx

```diff
@@ -0,0 +1,3 @@
+/*
```

Comment:
If used as test this is not necessary, but if used as a console example it could be

Reply:
The idea is to keep the example as closer as possible to the empty plugin.
If you prefer, I can remove it.